src/stores/PersonenKontextStore.spec.ts

```diff
@@ -175,4 +176,54 @@ describe('PersonenkontextStore', () => {
       expect(personenkontextStore.loading).toBe(false);
     });
   });
-});
\ No newline at end of file
+  describe('getAllPersonenuebersichte', () => {
+    it('should get All Personenuebersicht', async () => {
+      const mockResponse: DBiamPersonenuebersichtControllerFindPersonenuebersichten200Response = {
+        total: 0,
+        offset: 0,
+        limit: 0,
+        items: [
+          {
+            personId: 'string',
+            vorname: 'string',
+            nachname: 'string',
+            benutzername: 'string',
+            zuordnungen: [
+              {
+                sskId: 'string',
+                rolleId: 'string',
+                sskName: 'string',
+                sskDstNr: 'string',
+                rolle: 'string',
+              },
+            ],
+          },
+        ],
+      };
+      mockadapter.onGet('/api/dbiam/personenuebersicht').replyOnce(200, mockResponse);
+      const getAllPersonenuebersichtePromise: Promise<void> = personenkontextStore.getAllPersonenuebersichte();
```

Comment:
```suggestion
      const getAllPersonenuebersichtenPromise: Promise<void> = personenkontextStore.getAllPersonenuebersichten();
```
Bitte überall anpassen